lib/app/widgets/banners.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:banners/banners.dart';
+import 'package:flutter/material.dart';
+import 'package:schulcloud/app/app.dart';
+
+class OfflineBanner extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return BannerScaffold(
+      backgroundColor: Theme.of(context).errorColor.withOpacity(0.5),
+      body: Padding(
+        padding: EdgeInsets.all(8),
+        child: Text(context.s.app_error_noConnection),
+      ),
+    );
+  }
+}
+
+class TokenExpiredBanner extends StatelessWidget {
```

Comment:
We might want to include a sign-out-icon here directly so the user doesn't have to search for it